docs/indexing.md

```diff
@@ -197,7 +197,15 @@ above to do this)
 
 ### Swap in the new index
 
-First, update the index managers to have the new solr_collection values.
+Before you swap in the new index, make sure that the solr cloud is all working as expected:
+
+1. Open the solr console
```

Comment:
We should link to the connecting to the solr cloud portion of the doc `[Open the solr console](#accessing-the-solr-admin-ui)` I believe is the correct link.